docs/kyma/docs/017-details-local-installation.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Local installation
+type: Details
+---
+
+For better understanding of complex installation process, this guide describes it step by step.
+
+## Start local installation
+
+To fire local installation run following command:
+```
+./installation/cmd/run.sh
+```
+
+This script sets up default parameters, starts Minikube, builds Kyma-Installer, generates local configuration, creates `installer` Custom Resource and sets up Installer. Subsequent sections provide a detailed description of each step.
+
+### Installation parameters
+
+You can execute the `installation/cmd/run.sh` script with the following parameters:
+
+- `--skip-minikube-start` - it skips execution of the `installation/scripts/minikube.sh` script. See the "Start Minikube" section for more details.
```

Comment:
- `--skip-minikube-start` - it skips -> - `` `--skip-minikube-start` which skips`` (guidelines)
- "Start Minikube" ->` **Start Minikube** ` (guidelines)